src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -722,6 +722,19 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref Compo
                     }
                 }
 
+                if (typeWithAnnotationsOpt.HasType)
```

Comment:
>if (typeWithAnnotationsOpt.HasType) [](http://example.com/codeflow?start=16&length=35)

The other PR does this conditionally. Do you think  we should do this unconditionally? Perhaps ```typePredicate``` should be always required then?

Reply:
Added `visitCustomModifiers` based on [https://github.com/dotnet/roslyn/pull/58288](https://github.com/dotnet/roslyn/pull/58288).